feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/transaction/history/mixin/TransactionHistoryProvider.kt

```diff
@@ -68,7 +72,12 @@ class TransactionHistoryProvider(
         )
     }
 
-    override val state = domainState.map(::mapOperationHistoryStateToUi)
+    private val tokenFlow = walletInteractor.assetFlow(chainId, assetId)
+        .map { it.token }
```

Comment:
Its better to use `tokenUseCase` for that. `assetFlow` will trigger each time some asset updates in bd which produces a lot of extra triggers